bower.json

```diff
@@ -17,33 +17,29 @@
     "tests"
   ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^0.5",
-    "core-icons": "Polymer/core-icons#^0.5",
-    "paper-toggle-button": "Polymer/paper-toggle-button#^0.5",
-    "paper-fab": "Polymer/paper-fab#^0.5",
-    "paper-toast": "Polymer/paper-toast#^0.5",
-    "paper-dialog": "Polymer/paper-dialog#^0.5",
-    "core-collapse": "Polymer/core-collapse#^0.5",
-    "paper-spinner": "Polymer/paper-spinner#^0.5",
-    "paper-button": "polymer/paper-button#^0.5",
-    "paper-tabs": "Polymer/paper-tabs#^0.5",
-    "font-roboto": "Polymer/font-roboto#^0.5",
-    "core-toolbar": "Polymer/core-toolbar#^0.5",
-    "core-header-panel": "Polymer/core-header-panel#^0.5",
-    "paper-icon-button": "Polymer/paper-icon-button#^0.5",
-    "core-drawer-panel": "Polymer/core-drawer-panel#^0.5",
-    "core-pages": "Polymer/core-pages#^0.5",
-    "paper-progress": "polymer/paper-progress#^0.5",
-    "paper-shadow": "polymer/paper-shadow#^0.5",
-    "core-animated-pages": "polymer/core-animated-pages#^0.5",
-    "core-scaffold": "polymer/core-scaffold#^0.5",
-    "core-item": "polymer/core-item#^0.5",
-    "core-menu": "polymer/core-menu#^0.5",
-    "paper-menu-button": "polymer/paper-menu-button#^0.5",
-    "core-splitter": "polymer/core-splitter#^0.5",
-    "webcomponentsjs": "polymer/webcomponentsjs#^0.5"
-  },
-  "resolutions": {
-    "webcomponentsjs": "~0.5.5"
+    "polymer": "Polymer/polymer#~1.0.2",
+    "iron-flex-layout": "PolymerElements/iron-flex-layout#^1.0.0",
```

Comment:
It'd may be possible to replace most of these imports with:
```
"iron-elements": "PolymerElements/iron-elements#^1.0.0",
"paper-elements": "PolymerElements/paper-elements#^1.0.0"
```
but I haven't tried it -